core/src/main/java/com/microsoft/applicationinsights/extensibility/context/ContextTagKeys.java

```diff
@@ -185,7 +189,7 @@ public final String getApplicationId() {
     /**
      * @param value new value of ApplicationId property
      */
-    public final void setApplicationId(String value) {
```

Comment:
Why are we removing the public accessor? Does it break back compat?

Reply:
that shouldn't have been changed. 

Technically, these keys should not have setters. The keys are predetermined and are given special treatment when ingested; they key name shouldn't be mutable. I was thinking of doing this to the setters until I realized it's a bond generated file.